doctor_visits/delphi_doctor_visits/update_sensor.py

```diff
@@ -125,15 +126,19 @@ def update_sensor(
         (burn_in_dates >= startdate) & (burn_in_dates <= enddate))[0][:len(sensor_dates)]
 
     # handle if we need to adjust by weekday
-    params = Weekday.get_params(
-        data,
-        "Denominator",
-        Config.CLI_COLS + Config.FLU1_COL,
-        Config.DATE_COL,
-        [1, 1e5, 1e10, 1e15],
-        logger,
-    ) if weekday else None
-    if weekday and np.any(np.all(params == 0,axis=1)):
+    params = (
```

Comment:
if you want to get a smaller diff, you can probably revert the changes to this file, which will get it ignored by the linter.